stetho/src/main/java/com/facebook/stetho/inspector/elements/android/RealWindowDescriptor.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.view.View;
+
+import com.facebook.stetho.common.Accumulator;
+import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.android.window.WindowFetcher;
+import com.facebook.stetho.inspector.elements.android.window.WindowWrapper;
+
+import java.util.List;
+
+import static com.facebook.stetho.inspector.elements.android.ApplicationDescriptor.belongToActivity;
```

Comment:
This strategy often makes code way harder to review/read since I have to jump up to the imports to see that belongToActivity wasn't reimplementing in this file scope.  Consider what it is like reviewing code without the IDE :)